compiler/src/dotty/tools/dotc/decompiler/DecompilationPrinter.scala

```diff
@@ -24,8 +26,9 @@ class DecompilationPrinter extends Phase {
       var os: OutputStream = null
       var ps: PrintStream = null
       try {
+        implicit val codec = Codec.UTF8
         os = File(outputDir.fileNamed("decompiled.scala").path).outputStream(append = true)
-        ps = new PrintStream(os)
+        ps = new PrintStream(os, /*autoFlush*/false, "UTF-8")
```

Comment:
I would mimic the named argument syntax:
```suggestion
        ps = new PrintStream(os, /* autoFlush = */ false, "UTF-8")
```